### PR TITLE
samples: peripheral: 802154_phy_test: support nRF54L15

### PR DESCRIPTION
**Pull request by jciupis**
Enable 802154_phy_test sample for nRF54L15.

### CODE REVIEW

**Review by dawidprzybylo — APPROVED** [review]

**Comment by ankuns** [comment]
What about the table here? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/peripheral/802154_phy_test/README.html#requirements

what about changelog entry?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-changelog.html#peripheral-samples

**Comment by jciupis** [comment]
> What about the table here? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/peripheral/802154_phy_test/README.html#requirements

The contents of this table are generated automatically from the sample.yaml file. If you follow the documentation link posted above, the table already contains entry for nRF54L15 in this PR.

> what about changelog entry?
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-changelog.html#peripheral-samples

Added just now, to be checked if the generated documentation is correctly formatted.

**Review by ankuns — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -468,6 +468,12 @@ Peripheral samples
     * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
     * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
 
+* :ref:`802154_phy_test` sample:
+
+  * Added:
+
+    * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
```
Comment (FrancescoSer) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:475]:
```suggestion
  * Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
```

**Review by FrancescoSer — APPROVED** [review]
Approving but please implement the small suggestion

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (jciupis) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: Thank you for the suggestion, done.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -480,6 +480,10 @@ Peripheral samples
     * Support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
     * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
 
+* :ref:`802154_phy_test` sample:
+
+  * Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
```
Comment (annwoj) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:485]:
```suggestion
  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
```
Please update the link to redirect to our getting started page

**Review by annwoj — CHANGES_REQUESTED** [review]

**Review by annwoj — APPROVED** [review]

**Pull request merged**